docs/community/contributing.md

```diff
@@ -85,20 +84,13 @@ or Eventing and chat with other project developers. See also the [Knative Slack
 - [Stack Overflow questions](https://stackoverflow.com/questions/tagged/*knative*):
 Knative tagged questions and curated answers.
 
-## Community Meetup
+### Community Meetups
```

Comment:
Should we keep this as a top level way to contribute vs. under "learn and connect"? (I generally think of them as separate things, but don't have a strong opinion here, so either way is fine)

Reply:
I think the meetups are about learning and connecting, so I think this extra level is logical. (I may also have copied it, I'm not sure...)